spec/fixtures/hiera/hiera.yaml

```diff
@@ -0,0 +1,8 @@
+---
+:backends:
+  - yaml
+:yaml:
+  :datadir: 'spec/fixtures/hiera'
+:hierarchy:
+  - fqdn/%{fqdn}
+  - spectest/%{spectest}
```

Comment:
Better use common as default hierachy here, than you don't need to set the spectest fact in the tests as well.
(Repeated for better readability)